doc/nrf/known_issues.rst

```diff
@@ -497,14 +497,6 @@ NCSDK-5580: nRF5340 only supports SoftDevice Controller
 Bootloader
 ==========
 
-.. rst-class:: v1-5-0
```

Comment:
I think we should keep this for now, and then not add v1-6-0 when that arrives. This way customers can see what is fixed between versions etc.